pymongo/uri_parser.py

```diff
@@ -18,7 +18,13 @@
 import warnings
 import sys
 
-from urllib.parse import unquote_plus
+from urllib.parse import unquote_plus, unquote
+
+try:
+    from dns import resolver
+    _HAVE_DNSPYTHON = True
+except ImportError:
+    _HAVE_DNSPYTHON = False
```

Comment:
Please remove the unneeded _HAVE_DNSPYTHON stuff added in this PR. We already import it below.

Reply:
Done.